gateway/session_manager.go

```diff
@@ -309,13 +310,20 @@ func (l *SessionLimiter) ForwardMessage(r *http.Request, session *user.SessionSt
 
 }
 
+// RedisQuotaExceeded returns true if the request should be blocked as over quota.
 func (l *SessionLimiter) RedisQuotaExceeded(r *http.Request, session *user.SessionState, quotaKey, scope string, limit *user.APILimit, store storage.Handler, hashKeys bool) bool {
-	// Unlimited?
+	var (
```

Comment:
shall we make these values constants?

Reply:
no, local semantics only +0.01% on readability, may as well be unlimited